build.gradle

```diff
@@ -195,19 +195,15 @@ tasks.named('server').configure {
 }
 
 test {
-    // exclude code, which can not be instrumented correctly
-    jacoco {
-        excludes = ["com/gargoylesoftware/**"]
-    }
     // exclude tests which have own test tasks
     filter {
         excludeTestsMatching '*ContainerTest'
     }
     useJUnitPlatform()
```

Comment:
I think this would run the tests again. -> remove

Reply:
resolved in [48699f1](https://github.com/jenkinsci/ecu-test-execution-plugin/pull/118/commits/48699f12ef7ee4419425dece936414be7ee4dc2b)